staging/src/k8s.io/kubelet/pkg/apis/podresources/v1/api.pb.go

```diff
@@ -363,7 +363,7 @@ func (m *TopologyInfo) GetNodes() []*NUMANode {
 
 // NUMA representation of NUMA node
 type NUMANode struct {
-	ID                   int64    `protobuf:"varint,1,opt,name=ID,json=iD,proto3" json:"ID,omitempty"`
+	ID                   int64    `protobuf:"varint,1,opt,name=ID,proto3" json:"ID,omitempty"`
```

Comment:
would like an ack from @kubernetes/sig-node-api-reviews this doesn't impact any pod resource API serialization

Reply:
@SergeyKanzhelev @ehashman Can you please ack this change?

Reply:
I think @AlexeyPerevalov can help here! I also contributed to the feature, digging into the git history to restore the full context.

Reply:
Just to be clear, there is the same change on the`type Device struct` structure in two files above.

Reply:
My take, but please let's wait for @AlexeyPerevalov 's to confirm, is that we don't _need_ anything special for `NUMANode`.  It should behave like any other `podresources` API message. Not sure if some setting got lost somewhere. The intent was to just do what the rest of the proto file did. I'll research which settings `podresources` originally used. Also, thinking about it @RenaudWasTaken took care of the v1alpha1 -> v1 promotion, so he may want to comment here? I'm not sure something changed when moving to v1.

Reply:
if nothing in the kubelet path uses json proto serialization,  then this looks like a no-op change to me

Reply:
It didn't impact.
podresources is not only the one place, where I found json=iD. I also found it in libopenstorage an d deviceplugin. Probably, it's due to generator modifies the first letter (to lower case)

Reply:
right. none of those paths use json proto serialization, correct?

Reply:
> right. none of those paths use json proto serialization, correct?

AFAIK nothing uses the json proto